crmsh/bootstrap.py

```diff
@@ -1362,38 +1362,44 @@ def init_sbd():
             if not confirm("SBD is already configured to use %s - overwrite?" % (configured_dev)):
                 return
 
-        dev = ""
         dev_looks_sane = False
         while not dev_looks_sane:
-            dev = prompt_for_string('Path to storage device (e.g. /dev/disk/by-id/...), or "none"', r'none|\/.*', dev)
+            dev = prompt_for_string('Path to storage device (e.g. /dev/disk/by-id/...), or "none", use ";" as separator for multi path', r'none|\/.*')
             if dev == "none":
                 _context.diskless_sbd = True
                 init_sbd_diskless()
                 return
-            if not is_block_device(dev):
-                print("    That doesn't look like a block device", file=sys.stderr)
-                dev = ""
-            else:
-                warn("All data on {} will be destroyed!".format(dev))
-                if confirm('Are you sure you wish to use this device?'):
-                    dev_looks_sane = True
+            dev_list = dev.strip(';').split(';')
```

Comment:
Why this initial `strip`? To remove the last `;` if it's incorrectly set by the user, right?

Reply:
Yes, just for remove ';' like `/dev/sda1;`